waiter/src/waiter/core.clj

```diff
@@ -131,6 +131,7 @@
                               ["/service-description-builder" :state-service-description-builder-handler-fn]
                               ["/service-maintainer" :state-service-maintainer-handler-fn]
                               ["/statsd" :state-statsd-handler-fn]
+                              ["/tokens-watch-maintainer" :state-tokens-watch-maintainer]
```

Comment:
Can we use `/token-watch-maintainer` (singular) instead.
Also, a link to this page should be included in the `/state` output.